test/README

```diff
@@ -57,22 +57,22 @@ follows:
   .orientation[2] = n + 1
   .orientation[3] = n + 1
   .num_ranges     = n + 1
-  .ranges[i]      = i         (i = 0 ... n-1)
+  .ranges[i]      = i         (i = 0 ... n)
   .name           = <string representation of n+1>
   .enabled        = (n+1) % 2
 
 
-3. example_list_t
```

Comment:
The tests are actually not using `example_list_t` but `primitives_list_t` and there is no such type in the `test/types` directory.